frontend/vite.config.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types="vitest" />
+import {resolve} from 'path';
+import {defineConfig} from 'vite'
+import react from '@vitejs/plugin-react'
+import {browserslistToTargets} from "lightningcss";
+
+// https://vitejs.dev/config/
+export default defineConfig(({command, mode}) => {
+    let lib = 'student-ui';
+    let libName = 'AccordionBlock';
+    if (process.env.COMPONENT === 'studio-ui') {
+        lib = 'studio-ui';
+        libName = 'AccordionEditor';
+    }
```

Comment:
We build the student and studio UI separately since they use separate components and students don't need to load the whole of tinymce just to show an accordion.